cloud_formation/lambda/auto-build-test/auto-build-test.py

```diff
@@ -0,0 +1,227 @@
+# Copyright 2019 The Johns Hopkins University Applied Physics Laboratory
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+""" Lambda to launch ec2-instances """
+import boto3
+import os
+
+REGION = os.environ['AWS_REGION'] # region to launch instance leveraging lambda env var.
+AMI = 'ami-456b493a' # Ubuntu 16.04 AWS provided base image.
+INSTANCE_TYPE = 't2.micro' # instance type to launch.
+
+EC2 = boto3.client('ec2', region_name=REGION)
+
+def lambda_to_ec2(event, context):
+    """ Lambda handler taking [message] and creating a httpd instance with an echo. """
+
+    # bash script to run:
+    #  - boss-manage commands to build stack
+    #  - run tests using the instance's shell
+    #  - send all results to an S3 bucket
+    #  - send out completion SNS
+    #  - set to shutdown the instance automatically in 60 minutes.
+    init_script = """#!/bin/bash
+echo ' '
+echo 'echo "----------------------Init_Script----------------------'
+echo ' '
+
+set -e 
+
+#Install python3.5 and guide pip to it
+yes | apt-get update
+yes | ufw allow 22
+yes | apt-get install python3.5
+yes | apt-get install zip unzip
+yes | apt-get install jq
+yes | apt-get install python3-pip
+
+cd /home/ubuntu/
+
+# Create dummy .aws configs
+mkdir .aws
+cd .aws
+touch credentials
+touch config
+cd ..
+
+#Install git
+yes | apt-get install git
+
+# Compile Packer binary from source
```

Comment:
This comment should just say "Download Packer".

Reply:
Done